### DISCUSSION TITLE
Display recipe author/importer.

### DISCUSSION THREAD

**Discussion by instinkt900** [mealie-recipes/mealie#3790] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Recipes do not seem to display the user that imported or authored a given recipe.

### Please provide a concise description of the feature that would resolve your issue.

It would be useful to have a field somewhere that links to the user that imported the recipe.

### Please consider and list out some caveats or tradeoffs made in your design decision

None.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
It should be in the timeline. That's probably prominent enough IMO?

**Reply by instinkt900** [reply]
I did look at the timeline and it does show the avatar but doesn't seem to contain the name. If the users have similar or even the same avatar it's not obvious who the user is.

**Reply by michael-genson** [reply]
We should probably make the user clickable on the timeline

**Reply by instinkt900** [reply]
That would be ideal, but I'd settle for even just the username as text.

**Reply by boc-the-git** [reply]
Name in the text makes sense, it's what we do for meal planner logs

**Reply by michael-genson** [reply]
We can do both - we probably can't retroactively add the name to existing events
